File: benchmarks/fibonacci/fib.dart
int fib(int n) {
  if (n <= 1) return n;
  return fib(n - 1) + fib(n - 2);
}

void main() {
  print("Calculating fib(35)...");
  int result = fib(35);
  print("Result:\n$result");
}

File: benchmarks/fibonacci/fib.zig
const std = @import("std");

fn fib(n: i32) i32 {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

pub fn main() !void {
    std.debug.print("Calculating fib(35)...\n", .{});
    const result = fib(35);
    std.debug.print("Result:\n{d}\n", .{result});
}
